docker/cyclecloud8/scripts/cyclecloud_install.py

```diff
@@ -472,19 +477,20 @@ def configure_msft_apt_repos():
         f.write("deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ {} main".format(lsb_release))
 
     with open('/etc/apt/sources.list.d/cyclecloud.list', 'w') as f:
-        f.write("deb [arch=amd64] https://packages.microsoft.com/repos/cyclecloud {} main".format(lsb_release))
+        
+        f.write("deb [arch=amd64] https://packages.microsoft.com/repos/cyclecloud{'-insiders' if insiders_build else ''} {} main".format(lsb_release))
```

Comment:
I think this line fails (at least it does for me).   
`foo="deb [arch=amd64] https://packages.microsoft.com/repos/cyclecloud{'-insiders' if insiders_build else ''} {} main".format(lsb_release)
Traceback (most recent call last):
  File "<python-input-3>", line 1, in <module>
    foo="deb [arch=amd64] https://packages.microsoft.com/repos/cyclecloud{'-insiders' if insiders_build else ''} {} main".format(lsb_release)`
    
Might as well make it an f-string:
`>>> foo=f"deb [arch=amd64] https://packages.microsoft.com/repos/cyclecloud{'-insiders' if insiders_build else ''} {lsb_release} main"
>>> foo
'deb [arch=amd64] https://packages.microsoft.com/repos/cyclecloud-insiders 8.7.0 main'`